components/button/button.js

```diff
@@ -7,9 +7,19 @@ const { PropTypes } = React;
 /**
  * General Button component. Use when you need button or a link that looks like button
  */
-function Button({ children, mods = [], size, href, onClick, type, variation, disabled, dataAttrs = {} }) {
+function Button(props) {
+  const {
+    children,
+    size,
+    href,
+    onClick,
+    type,
+    variation,
+    disabled,
+    dataAttrs = {},
+  } = props;
   const restProps = getDataAttributes(dataAttrs);
-
+  const mods = props.mods ? props.mods.slice() : [];
```

Comment:
here is the docs for `mods`:
https://github.com/Travix-International/travix-ui-kit/pull/78/files/e6642dab2b515c3a613b7f0620ec86d6e3e41e4d#diff-22fbe4ecce9fc18f113d7f3df5fa1cc7R105 I guess the concept of mods is unclear: the mods property is for custom class selectors which developers need sometimes. And also we use it internally in components.

You can check how it works here https://travix-international.github.io/travix-ui-kit/#!/Button/1 there is an interactive playground.

The idea and the scope of this PR are to fix mutation of `mods` in all components, and not to change the way how they are work.